### PR TITLE
Updating get started next steps

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

Fixing the Next steps link [per this internal conversation](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1672940848329949). 

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/get-started/getting-started/building-your-first-project/build-your-first-models.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ keywords: [projects, create a project, dbt project, get started with Cloud]
 
 After setting up a sample project with a successful run, you can get to the fun part — [building models](/docs/build/sql-models)! You will learn how to take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.
 
+
```
Comment (runleonarun) [inline website/docs/docs/get-started/getting-started/building-your-first-project/build-your-first-models.md]:
```suggestion
```

File: website/snippets/tutorial-next-steps-setting-up.md [hunk 2]
```diff
@@ -1 +1 @@
-You can **[Continue the tutorial by building your first project](/building-your-first-project)**. You will learn and apply the fundamentals of dbt with models, tests, docs, and deploying your first job!
+You can Continue the tutorial by [building your first project](/get-started/getting-started/building-your-first-project/build-your-first-models). You will learn and apply the fundamentals of dbt with models, tests, docs, and deploying your first job!
```
Comment (nghi-ly) [inline website/snippets/tutorial-next-steps-setting-up.md]:
```suggestion
You can continue the tutorial by [building your first project](/docs/get-started/getting-started/building-your-first-project/build-your-first-models). You will learn and apply the fundamentals of dbt with models, tests, docs, and deploying your first job!
```

**Review by nghi-ly — COMMENTED** [review]
Thanks for fixing this, @runleonarun ! left one comment

**Review by nghi-ly — APPROVED** [review]
:shipit:

**Pull request merged**